### ISSUE TITLE
Some questions about the time expression task

### ISSUE THREAD

**Issue by weikang-wang** [Machine-Learning-for-Medical-Language/source-free-domain-adaptation#8]
Sorry ,I am not a native English speaker. So, what's the difference between type“Calendar-Interval” and type “Period”.

> Meantime, FBI agents and Metropolitan Police officers assigned to a joint terrorism task force here scanned the crowd of anti-abortion protesters at the annual March for Life on Capitol Hill, because Kopp has been either a participant in or arrested at this march in each of the **_last three years_**, according to another law enforcement official,
> 
> last three years 
> 
> years is "Calendar-Interval" type

> An ambitious expansion has left Magna with excess capacity and a heavy debt load as the automotive industry enters a downturn. The company has reported declines in operating profit in each of the **_past three years_**, despite steady sales growth.
> 
> years is "Calendar-Interval" type

> Mr. Obama said later at a news conference in Amman that he had spoken to both leaders over the **_past two years_** about how it was in the interests of both countries to restore normal relations.
> 
> years is "Period" type

And there are more cases in  both types, such as months .
I am very comfused.

**Comment by bethard** [comment]
The difference between Calendar-Interval and Period is whether the interval is aligned to a calendar year (i.e., starting in January and ending in December) or is the same length as a year, but not necessarily aligned to the calendar year. This is a hard call and is one of thing things that is challenging for both annotators and systems.

That said, I think the presence of "each of the" in your first two cases, at least to my ear, suggests that they're talking about calendar years, while in the last case, where that is not present, I read it as the two-year interval that ends on the day of the news conference (so, not aligned to a calendar year).

**Comment by weikang-wang** [comment]
Thank you，and I have another question.

> seasonal trends, seasonal market, Seasonal rains

Is the word "seasonal" in Calendar-Interval type ？

**Comment by bethard** [comment]
No, the word "seasonal" is a Season-Of-Year.

**Comment by weikang-wang** [comment]
> per hour, per day, per month, per year

So these are Calendar-Interval type, isn't it ?

**Comment by bethard** [comment]
Yes, that's correct. In "per month", the "month" would be annotated as a Calendar-Interval. Similarly for the others.

**Comment by weikang-wang** [comment]
Thanks，and last question

> recent three-year average 
> the coming year

Are two "year " “Calendar-Interval” or type “Period” ?

**Comment by bethard** [comment]
The second is usually a Calendar-Interval. The first is a harder call without more context. If the averages are averages over the entire year, then that should probably also be a Calendar-Interval.
